Specs/NumberWithUnit/French/CurrencyModel.json

```diff
@@ -624,6 +624,14 @@
         "value": "63,45",
         "unit": "Dollar"
       }
+    },
+    {
+      "Text": "cent",
```

Comment:
Cent in 8 pour cent should not be extracted in this case.
Let's disable this test with notsupported and open a new issue to fix it.